packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -131,6 +154,71 @@ export default class QuanticSort extends LightningElement {
     this.hasInitializationError = true;
   }
 
+  /**
+   * Sets the error when custom sort options have an invalid configuration.
+   */
+  setSortOptionsConfigurationError() {
+    this.errorMessage = `Custom sort options configuration is invalid.`;
+  }
+
+  /**
+   * Generates the schemas for the validation of the sort options with Bueno.
+   */
+  generateSchemas() {
+    this.headless = getHeadlessBundle(this.engineId);
```

Comment:
`this.headless = getHeadlessBundle` is already execute in the `initialize` method, it would be nice not to have to repeat it?

Reply:
yeah no this I meant to remove, I moved it to the initialize but forgot to remove that one :p